dts/arm/st/f0/stm32f030Xc.dtsi

```diff
@@ -8,7 +8,17 @@
 #include <st/f0/stm32f030.dtsi>
 
 / {
+	sram0: memory@20000000 {
+		reg = <0x20000000 DT_SIZE_K(8)>;
```

Comment:
According to ST, RAM is 32K for these devices.

![image](https://user-images.githubusercontent.com/25011557/79982308-64467c80-84a6-11ea-97d2-d140958e8dbb.png)

Reply:
Fixed.